### DISCUSSION TITLE
feature

### DISCUSSION THREAD

**Discussion by riyanwe** [Zero6992/chatGPT-discord-bot#79] [Q&A]
can't you add an embed in reply to chatgpt messages?

**Comment by Zero6992** [comment]
I'm not really sure what the difference is between an embed reply and a regular reply, or what kind of better user experience it brings

I'll do some research and see what I can find out anyway, thank you for the suggestion though

**Comment by JA1E0** [comment]
> can't you add an embed in reply to chatgpt messages?

It works.
